daemon/api.go

```diff
@@ -2352,6 +2360,80 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	}, nil)
 }
 
+type postUserRemoveData struct {
+	Username string `json:"username"`
+}
+
+func postRemoveUser(c *Command, r *http.Request, user *auth.UserState) Response {
+	_, uid, _, err := postRemoveUserUcrednetGet(r.RemoteAddr)
+	if err != nil {
+		return BadRequest("cannot get ucrednet uid: %v", err)
```

Comment:
Nitpick: this is a bit technical, perhaps just `cannot get peer credentials`?

Reply:
Agree that message is cryptic, but it's consistent with postCreateUser(). Should we change in both places?

Reply:
I will defer to @chipaca but I would say yes unless you prefer the original message.

Reply:
Checking the ucrednet code, it seems that the original error message is accurate: it's actually retrieving the UID from the remote address and not credentials.

Reply:
however, to be clear, `create-user` is rather crufty and not how we would've approached things today. In fact it's in dire need of a refactor... so "create-user does it this way" is not a very good metric ... but it's on us, not on you, to fix these things. Refactoring is harder than coding from scratch :-)